globals/strings.gd

```diff
@@ -684,7 +696,14 @@ func get_effect_text(effect):
 				prevent_life_str = " (Can't lose life)"
 			text += "Down %s Holomem%s%s." % [target_str, required_damage_str, prevent_life_str]
 		"draw":
-			text += tr("Draw %s.") % [effect["amount"]]
+			var amount = effect.get("amount", "")
+			var draw_to_hand_size_str = ""
+			var draw_to_hand_size = effect.get("draw_to_hand_size")
+			if draw_to_hand_size:
+				if amount != "":
+					draw_to_hand_size_str += ", "
+				draw_to_hand_size_str += "until you have %s cards in hand" % draw_to_hand_size
```

Comment:
can you make this a completely separate string and tr() it please?
so tr("Draw until...")
You can just put english in all slots in the sheet if you want, but let's at least set this up since Draw was already tr()'d